node/connection_base.js

```diff
@@ -27,6 +27,7 @@ var EventEmitter = require('./lib/event_emitter');
 
 var errors = require('./errors');
 var States = require('./reqres_states');
+var Operations = require('./operations');
```

Comment:
Operations are CallRequest / CallResponse pairs. We eliminated Operation previously, in favor of using the Request object as a proxy for the corresponding Response. Probably should call this Requests to be consistent with that decision.